lib/local-links-manager/import/csv_downloader.rb

```diff
@@ -26,4 +33,10 @@ def downloaded_csv
 
     response.body
   end
+
+  def name_converter(conversions)
```

Comment:
I don't know that `name_converter` is the most intention revealing name.  Not that I think I have a better one...  Not too keen on overusing `header` either, so it's not like `header_converters` would be better.